e2e_tests/src/e2e_indexer/e2elive.py

```diff
@@ -131,6 +137,8 @@ def main():
                     with open(p) as nf:
                         for line in nf:
                             logger.error("   {}".format(line))
+        if found is False:
```

Comment:
It seems that when arriving to this line it will always be the case that `found is False`. Did you intend to set `found = True` after line 135 ?